readme.md

```diff
@@ -162,13 +202,13 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 </html>
 ```
 
-> Your answer...
+Close the DOCTYPE, close the title, and put an '=' after the opening yield party hats
 
 ## Question 13
 
 Of the three options below, which is the most "correct" way of organizing the files that make up an Angular app, as used in class? Why is this option considered "better" than the other two?
 
-> Your answer...
+> B?
```

Comment:
That's correct, it's closest to what we did in class, and the conventions around structuring angular apps.